### PR TITLE
FIX: ensures only one msg-action on mobile

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: assets/javascripts/discourse/components/chat-message.js [hunk 1]
```diff
@@ -639,7 +641,7 @@ export default Component.extend({
     this.notifyPropertyChange("emojiReactions");
 
     if (this.site.mobileView) {
-      this.toggleProperty("isHovered");
+      this.set("isHovered", false);
```
Comment (jjaffeux) [inline assets/javascripts/discourse/components/chat-message.js:644]: Toggle seems un-necessary here, if we reacted we want to close the menu 100%.

File: assets/javascripts/discourse/components/chat-message.js [hunk 2]
```diff
@@ -230,7 +231,8 @@ export default Component.extend({
   @action
   handleClick() {
     if (this.site.mobileView) {
-      this.toggleProperty("isHovered");
+      this.onHoverMessage && this.onHoverMessage(this.message);
```
Comment (CvX) [inline assets/javascripts/discourse/components/chat-message.js:234]: 🏌️ ⛳

```suggestion
      this.onHoverMessage?.(this.message);
```

File: assets/javascripts/discourse/components/chat-live-pane.js [hunk 3]
```diff
@@ -1222,6 +1223,14 @@ export default Component.extend({
     }
   },
 
+  @action
+  onHoverMessage(message) {
+    this.set(
+      "hoveredMessageId",
+      message?.id && message.id !== this.hoveredMessageId ? message.id : null
```
Comment (markvanlan) [inline assets/javascripts/discourse/components/chat-live-pane.js:1230]: This is difficult to read but.. if it works that's ok

**Review by markvanlan — APPROVED** [review]

File: assets/javascripts/discourse/components/chat-live-pane.js (hunk 3 above)
Comment (jjaffeux) [reply assets/javascripts/discourse/components/chat-live-pane.js:1230]: yep there are various ways to write it, I finally stopped on this one, I think that's okayish.

**Pull request merged**